prow/scripts/create-dns-entry.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+#In order to run this script you need to use a service account with DNS Administrator role
+
+set -o errexit
+
+if [ -z "$IP_ADDRESS" ]; then
+    echo "\$IP_ADDRESS is empty"
+    exit 1
+fi
+
+if [ -z "$PROJECT" ]; then
+    echo "\$PROJECT is empty"
+    exit 1
+fi
+
+if [ -z "$DNS_ZONE" ]; then
+    echo "\$DNS_ZONE is empty"
+    exit 1
+fi
+
+if [ -z "$DOMAIN" ]; then
+    echo "\$DOMAIN is empty"
+    exit 1
+fi
+
+RANDOM_STRING=$(cat /dev/urandom | env LC_CTYPE=C tr -dc a-z0-9 | head -c 16; echo)
+DNS_NAME="pull-${RANDOM_STRING}.${DOMAIN}"
```

Comment:
this script will be used only for pull requests?

Reply:
could you provide me an example what will be DNS name?

Reply:
In my opinion, DNS_NAME should be given as input to this script. It's up to the caller to determine the name.